reoptjl/views.py

```diff
@@ -855,7 +855,10 @@ def queryset_for_summary(api_metas,summary_dict:dict):
     )
     if len(tariffOuts) > 0:
         for m in tariffOuts:
-            summary_dict[str(m.meta.run_uuid)]['year_one_savings_us_dollars'] = (m.year_one_energy_cost_before_tax_bau + m.year_one_demand_cost_before_tax_bau + m.year_one_fixed_cost_before_tax_bau + m.year_one_min_charge_adder_before_tax_bau) - (m.year_one_energy_cost_before_tax + m.year_one_demand_cost_before_tax + m.year_one_fixed_cost_before_tax + m.year_one_min_charge_adder_before_tax)
+            if len(tariffInputs) > 0:
+                summary_dict[str(m.meta.run_uuid)]['year_one_savings_us_dollars'] = (m.year_one_energy_cost_before_tax_bau + m.year_one_demand_cost_before_tax_bau + m.year_one_fixed_cost_before_tax_bau + m.year_one_min_charge_adder_before_tax_bau) - (m.year_one_energy_cost_before_tax + m.year_one_demand_cost_before_tax + m.year_one_fixed_cost_before_tax + m.year_one_min_charge_adder_before_tax)
```

Comment:
Good points.  the "us_dollars" thing also seems like legacy v2 naming convention - but maybe we kept it instead of updating the web tool. If we changed it to `year_one_bill_before_tax` - `..._bau`, would that include the export benefit? Or do we still need to account for that separately?

Reply:
We would still need to account for that separately. The `year_one_bill_before_tax` does not account for the value of exports.

Reply:
Updated to use `year_one_bill`